pkg/portal/cluster/clusteroperators.go

```diff
@@ -25,17 +36,38 @@ func clusterOperatorsInformationFromOperatorList(operators *configv1.ClusterOper
 	}
 
 	for _, co := range operators.Items {
-		var Available = configv1.ConditionUnknown
+		var available = configv1.ConditionUnknown
+		var progressing = configv1.ConditionUnknown
+		var degraded = configv1.ConditionUnknown
 
+		var conditions []OperatorCondition
 		for _, cnd := range co.Status.Conditions {
 			if cnd.Type == "Available" {
-				Available = cnd.Status
+				available = cnd.Status
+			} else if cnd.Type == "Progressing" {
+				progressing = cnd.Status
+			} else if cnd.Type == "Degraded" {
+				degraded = cnd.Status
+			}
```

Comment:
a switch case would be better here, also, please use the constants provided in configv1

Reply:
Fixed